### PR TITLE
[MVS-462] Create endpoint for Observation

### PR DESCRIPTION
**Pull request by axn9t9**
## :flipper: [JIRA Sprint Card]
https://mass-immunization-system.atlassian.net/browse/MVS-462

## :newspaper: [Work Description]
Defined template for incoming request data and add logic to structure the incoming request data into the FHIR standard.

## :vertical_traffic_light: [Testing/QA Notes]
- Run docker containers
- Create preliminary FHIR resources (referenced FHIR resources must exist)
- Send a POST request to /Observation with the request data formatted according to template in models directory.
- Ensure the response data is the equivalent FHIR resource, with all of the provided information.

### CODE REVIEW

File: node_app/models/Observation.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+/*
+Observation {
+  status : enum (final, amended, corrected)
+  category : enum (vital-signs, procedure, exam)
+  code : enum (?)
```
Comment (dmcook5208) [inline node_app/models/Observation.js:5]: I also don't know which enum values we need to provide.  Who can help us with that?

**Review by dmcook5208 — APPROVED** [review]

File: node_app/models/Observation.js (hunk 1 above)
Comment (rrucker3) [reply node_app/models/Observation.js:5]: What will these Observations represent? What clinical measurements are taken during the workflow. If it's blood pressure, for example, we can use a SNOMED code to represent that: https://snomedbrowser.com/Codes/Details/75367002

Comment (axn9t9) [reply node_app/models/Observation.js:5]: I'm not sure. The example valueset from hl7 has over a thousand codes

Comment (rrucker3) [reply node_app/models/Observation.js:5]: @dmcook5208 how does the adverse event documentation work? How will we allow vaccinators to document these?

Comment (dmcook5208) [reply node_app/models/Observation.js:5]: In the Immunization resource, there is this:
image

Right now, we have a page in the POD app for documenting adverse reactions...but this is something we should discuss with Waukesha county.  Anything a user documents in our app will also need to be documented in VAERS, so I'm not sure how beneficial this feature will really be at this point (as it is just duplicated documentation).  Here is what our "Adverse Reactions" page would currently look like - it mimics the VAERS form:
image

**Pull request closed without merging**